### ISSUE TITLE
schema user creation

### ISSUE THREAD

**Issue by sriram9707** [aws-quickstart/quickstart-sap-hana#26]
Does schema user get created with the quickstart ? We dont see a schema user  and if not, can we create it maually from hana studio ?

**Comment by gguifelixamz** [comment]
No, schema user doesn't get created and it only creates the standard users that are default to the SAP HANA installation. Example, SYSTEM user.
